google/oauth2/flow.py

```diff
@@ -54,12 +54,22 @@
 """
 
 import json
+import logging
+import webbrowser
```

Comment:
If by "really doing this" you mean:

1. Using `webbrowser`, then yes, I think it's important if we want to be on parity with what oauth2client offered.
2. Having this code period - If you feel like this code shouldn't be here, by all means I'm happy to have that conversation. We have a lot of options, so if you feel it doesn't belong in `google-auth` proper let's talk.